src/content/app/genome-browser/services/browser-bookmarks/browserBookmarksStorageService.ts

```diff
@@ -15,29 +15,31 @@
  */
 
 import storageService, {
-  StorageServiceInterface
+  type StorageServiceInterface
 } from 'src/services/storage-service';
 
-import { PreviouslyViewedObject } from 'src/content/app/genome-browser/state/track-panel/trackPanelSlice';
+import type { PreviouslyViewedObject } from 'src/content/app/genome-browser/state/browser-bookmarks/browserBookmarksSlice';
 
 export enum StorageKeys {
-  BOOKMARKS = 'trackPanel.bookmarks',
-  PREVIOUSLY_VIEWED = 'trackPanel.previouslyViewedObjects'
+  BOOKMARKS = 'genomeBrowser.bookmarks',
+  PREVIOUSLY_VIEWED = 'genomeBrowser.previouslyViewedObjects'
 }
 
-export class TrackPanelStorageService {
+export class BrowserBookmarksStorageService {
   private storageService: StorageServiceInterface;
 
   public constructor(storageService: StorageServiceInterface) {
     this.storageService = storageService;
   }
 
-  public getBookmarks(): { [genomeId: string]: PreviouslyViewedObject[] } {
+  public getBookmarks(): {
+    [genomeId: string]: [];
```

Comment:
Why do we need to change `PreviouslyViewedObject` to []?

Reply:
We don't know the shape of the object for bookmarks so unfortunately can't use a non-generic type for it

Reply:
I am confused by our use of the word "bookmark".

We said, a long time ago, that we expected to have two types of bookmarks:
- the automatically generated "previously viewed objects"
- the bookmarks that a user can create manually.

So far, we have only been dealing with previously viewed objects. But the method, I see, has the word "bookmarks" in its name. Is this going to be confusing?

The `{ [genomeId: string]: []; }` is an incorrect type; it sets the value to the type `never[]`.

Reply:
This is because this method is for the second type and it was explicitly called as bookmarks then. Also we had this method inside the `TrackPanelStorageService`, so less confusion than with having it inside `BrowserBookmarksStorageService`. Or at least I think that's close to what the thinking was back then.

Now that it is confusing, should we change this to `getUserBookmarks`? Open to suggestions for better names than this. As for the type should it then be `{ [genomeId: string]: unknown[]; }`?

Reply:
Added comments above the two functions and renamed them ✅